app/Models/Entry.php

```diff
@@ -490,18 +490,18 @@ public static function getContentByParsing($url, $path, $attributes = array(), $
 
 	public function loadCompleteContent($force = false) {
 		// Gestion du contenu
-		// On cherche à récupérer les articles en entier... même si le flux ne le propose pas
+		// On cherche à récupérer les articles en entier même si le flux ne le propose pas
```

Comment:
As I said in another PR recently, it might be preferable to change lines like this to English if they're being changed anyway.

Reply:
Agreed